hw_diag/utilities/balena_supervisor.py

```diff
@@ -56,11 +56,24 @@ def get_device_status(self, key_to_return) -> str:
         LOGGER.info("Retrieving device status using Balena supervisor.")
 
         response = self._make_request('GET', '/v2/state/status')
-        if response is None or response.ok is False:
-            LOGGER.error("Device status request failed.")
-            raise Exception("Device status request failed")
+
+        error_msg = ""
```

Comment:
nit: False or None might be more clear

Reply:
yeah but this empty string is usually set as start value so if we need to concatenate multiple  error messages into a string later it becomes easier instead of needing to convert None of False to str first.